containers/ddev-router/Dockerfile

```diff
@@ -6,22 +6,50 @@ ENV DEBIAN_FRONTEND noninteractive
 ENV DOCKER_GEN_VERSION 0.7.4
 ENV DOCKER_HOST unix:///tmp/docker.sock
 
+# TARGETPLATFORM is Docker buildx's target platform (e.g. linux/arm64), while 
+# BUILDPLATFORM is the platform of the build host (e.g. linux/amd64)
+ARG TARGETPLATFORM
+ARG BUILDPLATFORM
+
+SHELL ["/bin/bash", "-c"]
+
 RUN apt-get -qq update && \
     apt-get -qq install --no-install-recommends --no-install-suggests -y \
         ca-certificates certbot curl less python-certbot-nginx procps telnet vim wget && \
     apt-get autoremove -y && \
     apt-get clean -y && \
 	rm -rf /var/lib/apt/lists/*
 
-ADD https://github.com/jwilder/forego/releases/download/v0.16.1/forego /usr/local/bin/forego
+RUN if [[ $TARGETPLATFORM == "linux/arm64" ]]; then \
+        # jwilder/forego doesn't have an arm64 download so we resort to the original forego repo
+        wget -q https://bin.equinox.io/c/ekMN3bCZFUn/forego-stable-linux-arm64.tgz && \
```

Comment:
As I studied forego for the cert/LetsEncrypt PR I started to understand that it is just another way to approach the problems we solve with supervisord in ddev-webserver. IMO we should switch to using supervisord. I'm just mentioning this in passing because I note that you hit this problem. No action is required.